definitions/ext-unix_host/golden_metrics.yml

```diff
@@ -0,0 +1,26 @@
+cpuUtilizationPercent:
+  title: CPU Utilization Percentage
+  displayAsValue: true
+  query:
+    select: 100-latest(cpu.idle)
+    from: '`unixMonitor:Stats`'
+    facet: hostname
+    eventId: entity.guid
+
+memoryUtilizationPercent:
+  title: Memory Utilization Percentage
+  displayAsValue: true
+  query:
+    select: (latest(memory.total) - latest(memory.free))/latest(memory.total) * 100
+    from: '`unixMonitor:Stats`'
+    facet: hostname
+    eventId: entity.guid
+
+diskUtilizationPercent:
+  title: Disk Utilization Percentage
+  displayAsValue: true
+  query:
+    select: max(percentUsed)
+    from: '`unixMonitor:Disk`'
+    facet: hostname, mountedOn 
```

Comment:
@srvaroa - is multi-facet supported on golden metrics yet?

Reply:
@naxhh you're more up to date than me on this

Reply:
@thezackm @naxhh tells me that it's still not supported, but no problem merging this as it is (it will not work straight away, but it's good to give us a first case to verify as soon as we complete that work).

Reply:
@thezackm We support this in the definition but is still not being used in Golden metrics (WIP)

Check: https://github.com/newrelic-experimental/entity-synthesis-definitions/blob/main/docs/golden_metrics.md#defining-golden-metrics

So when this is implemented this will convert into: `FACET entity.name, hostname, mountedOn`
Since you are using hostname as the `entity.name` you probably want this to look like:

`facet: mountedOn` the facet field is now "extra facets" in the same way we use "where" as extra where aside of `entity.guid IN ....`

hope that makes sense!

Reply:
perfect; I'm good with the changes and ready to go; thanks for all the support!

Reply:
ah, that's right @naxhh ! I forgot about that change since this branch was first made; let me take one last look through the golden and summary to validate things

Reply:
ok; ready to merge

@sschwartzman has mentioned we will want to look at a dashboard later, but that can be added outside of this workflow